admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -98,30 +101,35 @@ rhsub_pass: password <16>
 rhsub_pool: pool-id <17>
 endif::[]
 ----
-<1> The name of the existing cluster
-<2> The region the existing cluster is currently running in
-<3> Disable the creation of a VPC
-<4> The existing VPC name that the cluster is running in
-<5> The name of one subnet the existing cluster is running in
-<6> Disable the creation of security groups
-<7> The key name to use for SSH access
-<8> The base AMI image (https://www.redhat.com/en/technologies/cloud-computing/cloud-access)
-<9> Disable the creation of an S3 bucket
-<10> The security group name
-<11> The VPC subnets the existing cluster is running in
+<1> Provide the name of the existing cluster.
+<2> Provide the region the existing cluster is currently running in.
+<3> Specify `False` to disable the creation of a VPC.
+<4> Provide the existing VPC name that the cluster is running in.
+<5> Provide the name of a subnet the existing cluster is running in.
+<6> Specify `False` to disable the creation of security groups.
+<7> Provide the AWS key name to use for SSH access.
+<8> Provide the AMI image ID to use as the base for the primed image.
+See link:https://www.redhat.com/en/technologies/cloud-computing/cloud-access[Red Hat® Cloud Access].
+<9> Specify `False` to disable the creation of an S3 bucket.
+<10> Provide the security group name.
+<11> Provide the VPC subnets the existing cluster is running in.
+<12> Specify `overlay2` as the Docker storage type.
+<13> Specify the mount point for LVM and the *_/var/lib/docker_* directory, 
+which is the block device that the *_build_ami_* playbook creates.
 ifdef::openshift-enterprise[]
-<12> Use overlay2 storage type for docker
-<13> Mountpoint for LVM and /var/lib/docker; block device created by build_ami playbook
-<14> Enable gquota on / filesystem when using Red Hat Cloud
-<15> Red Hat subscription-manager email address
-<16> Red Hat subscription-manager password
-<17> Red Hat subscription-manager Pool ID
+<14> If you use Red Hat Cloud, set this parameter value to `true` to enable
+`gquota` on the file system.
+<15> Specify an email address that for a Red Hat account with an active
+{product-title} subscription.
+<16> Specify the password for the Red Hat account
+<17> Specify a pool ID for an {product-title} subscription. You can use the same
+pool ID that you used when you created your cluster.
 endif::[]
 
-. Run the AWS-specific playbook to generate a primed image
+. Run the *_build_ami.yml_* playbook to generate a primed image:
 +
 ----
-# ansible-playbook -i inventory \
+# ansible-playbook -i <path_to_inventory_file> \
```

Comment:
I don't know how we spell these examples out in the existing docs, but when specifying paths I tend to go with: `'/path/to/inventory'`, the presence of the `'/'` enforces that it is a path.